src/fmu/dataio/export/rms/inplace_volumes.py

```diff
@@ -23,6 +23,9 @@
 
 _logger: Final = null_logger(__name__)
 
+
+TABLE_INDEX_COLUMNS: Final = ("ZONE", "REGION", "FACIES", "LICENCE")
```

Comment:
Maybe just make this private in the unlikely event some user imports it for some reason?